### PR TITLE
WIP: Agile post

### PR DESCRIPTION
**Pull request by rstiles**
@adiyamo

### CODE REVIEW

**Comment by hilton** [comment]
Personally, I like the brutal imagery: for me that's what adds feeling to the article. I also like the storytelling style of the initial period of enthusiasm and thinking it was easy, followed by the challenges and darkness. However, I think something to balance that missing. I'd like it to have a more optimistic conclusion, either about subsequent success or a new understanding about what's going to be better in the future.

In the text, _Scrum_ should always be capitalised. Also, I suggest breaking up the screenful of text with two or three headings.

File: _posts/job-title-troublemaker.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Job Title: Troublemaker"
```
Comment (hilton) [inline _posts/job-title-troublemaker.md]: _Troublemaker_ sounds somewhat negative - what about something that picks up on _asking difficult questions_?

**Comment by TimKam** [comment]
@hilton could you do a final review? :-)

**Comment by hilton** [comment]
Sure - I should have time to do that tomorrow or Wednesday.

File: _posts/2017-12-13-scrum-master.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+If I could go back in time and give my past self some advice, it would be this: do not go and try to sell an idea. Instead, go out and ask the uncomfortable questions. Try to make people think. People may dislike it, but that is fine. That is where my role as a troublemaker comes in. Some people might actually dislike the questions so much that it will motivate them to discuss it.
+
+When obstacles have been made painfully obvious ,  people cannot work around them anymore. Sometimes they might decide it's simpler to work around an obstacle than to conquer it, but at least they will have an awareness of the obstacle. However, most of the time people will be motivated to drive change. That is my actual day-to-day-work: seeing the impediments that teams and companies face and talking about them. A lot.
```
Comment (AnimiVulpis) [inline _posts/2017-12-13-scrum-master.md]: Just went over the article (👍 ) and noticed
a `word`, followed by a `space` followed by a `,` followed by **two** `spaces`.

Is this a mistake? Or some arcane text formatting that I don't get?

Comment (hilton) [reply _posts/2017-12-13-scrum-master.md]: Weird mistake, fixed now. Looks like some weird characters that didn't show in my editor.

**Pull request merged**